style.css

```diff
@@ -307,6 +308,140 @@ main {
   color: #fff;
 }
 
+.modal {
```

Comment:
- Good work implementing some changes requested by the previous reviewer to make your website responsive. But there are some sizes in which you need to work. Kindly check your design after 1368px of weight since after this the design lost responsiveness

![image](https://user-images.githubusercontent.com/93630700/223744032-5b1252ae-1569-4ef0-b1f3-d69eba5e1b58.png)